db/sql/index.js

```diff
@@ -12,3 +12,57 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 'use strict';
+
+const QueryFile = require('pg-promise').QueryFile;
+const path = require('path');
+
+// Check if it is a packaged/prod application:
+const isPackaged = __filename.endsWith('app.js');
```

Comment:
So as per my understanding... Since the packaged building, concat all files together to `app.js` so this will work. But if we change the packaged file name, we will have to update this as well.

Reply:
@vitaly-t I believe we should put a comment on this line 

https://github.com/LiskHQ/lisk/blob/c387e94b2a4241eec360acd05d657dfdcb52ee85/Gruntfile.js#L44

So if someone later try to change it, he should also change the file here.

Reply:
Yes, correct, and it is done - comment added ;)